sysl2/sysl/parse.go

```diff
@@ -339,12 +339,12 @@ func infer_types(mod *sysl.Module, appName string) {
 			logrus.Warnf("view %s expression should be of type transform", viewName)
 			continue
 		}
-		infer_expr_type(mod, appName, view.Expr, true, 0)
+		inferExprType(mod, appName, view.Expr, true, 0)
 	}
 }
 
 func postProcess(mod *sysl.Module) {
-	var appNames []string
+	appNames := make([]string, 0, len(mod.Apps))
```

Comment:
hmm back to make() now? why?

Reply:
It's recommended when the final size is known in advance. I was quite impressed that one of the linters picks up on these opportunities.